[2022-10-26 15:14:33,681] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dbt_dim.eventos_dim scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 15:14:33,696] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dbt_dim.eventos_dim scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 15:14:33,697] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 15:14:33,697] {taskinstance.py:1357} INFO - Starting attempt 5 of 6
[2022-10-26 15:14:33,697] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 15:14:33,717] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): eventos_dim> on 2022-09-01 00:00:00+00:00
[2022-10-26 15:14:33,726] {standard_task_runner.py:52} INFO - Started process 884 to run task
[2022-10-26 15:14:33,733] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dbt_dim', 'eventos_dim', 'scheduled__2022-09-01T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dim.py', '--cfg-path', '/tmp/tmpzo9xh5pk', '--error-file', '/tmp/tmp8xmovysz']
[2022-10-26 15:14:33,735] {standard_task_runner.py:80} INFO - Job 27: Subtask eventos_dim
[2022-10-26 15:14:33,808] {task_command.py:369} INFO - Running <TaskInstance: dbt_dim.eventos_dim scheduled__2022-09-01T00:00:00+00:00 [running]> on host 4af2e895db31
[2022-10-26 15:14:33,938] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dbt_dim
AIRFLOW_CTX_TASK_ID=eventos_dim
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=5
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-01T00:00:00+00:00
[2022-10-26 15:14:33,939] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-10-26 15:14:33,940] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /opt/dbt/oly_dbt && dbt run --select +eventos_dim']
[2022-10-26 15:14:33,951] {subprocess.py:85} INFO - Output:
[2022-10-26 15:14:37,967] {subprocess.py:92} INFO - [0m15:14:37  Running with dbt=1.3.0
[2022-10-26 15:14:38,088] {subprocess.py:92} INFO - [0m15:14:38  Unable to do partial parsing because a project config has changed
[2022-10-26 15:14:39,569] {subprocess.py:92} INFO - [0m15:14:39  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2022-10-26 15:14:39,569] {subprocess.py:92} INFO - There are 1 unused configuration paths:
[2022-10-26 15:14:39,570] {subprocess.py:92} INFO - - models.oly_dbt.fact_tables
[2022-10-26 15:14:39,570] {subprocess.py:92} INFO - 
[2022-10-26 15:14:39,617] {subprocess.py:92} INFO - [0m15:14:39  Found 2 models, 0 tests, 0 snapshots, 0 analyses, 289 macros, 0 operations, 0 seed files, 0 sources, 0 exposures, 0 metrics
[2022-10-26 15:14:39,620] {subprocess.py:92} INFO - [0m15:14:39
[2022-10-26 15:14:39,892] {subprocess.py:92} INFO - [0m15:14:39  Concurrency: 4 threads (target='dev')
[2022-10-26 15:14:39,893] {subprocess.py:92} INFO - [0m15:14:39
[2022-10-26 15:14:39,898] {subprocess.py:92} INFO - [0m15:14:39  1 of 1 START sql table model public.eventos_dim ................................ [RUN]
[2022-10-26 15:14:40,159] {subprocess.py:92} INFO - [0m15:14:40  1 of 1 OK created sql table model public.eventos_dim ........................... [[32mSELECT 765[0m in 0.26s]
[2022-10-26 15:14:40,233] {subprocess.py:92} INFO - [0m15:14:40
[2022-10-26 15:14:40,234] {subprocess.py:92} INFO - [0m15:14:40  Finished running 1 table model in 0 hours 0 minutes and 0.61 seconds (0.61s).
[2022-10-26 15:14:40,269] {subprocess.py:92} INFO - [0m15:14:40
[2022-10-26 15:14:40,271] {subprocess.py:92} INFO - [0m15:14:40  [32mCompleted successfully[0m
[2022-10-26 15:14:40,272] {subprocess.py:92} INFO - [0m15:14:40
[2022-10-26 15:14:40,273] {subprocess.py:92} INFO - [0m15:14:40  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-10-26 15:14:42,099] {subprocess.py:96} INFO - Command exited with return code 0
[2022-10-26 15:14:42,241] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dbt_dim, task_id=eventos_dim, execution_date=20220901T000000, start_date=20221026T151433, end_date=20221026T151442
[2022-10-26 15:14:42,296] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-26 15:14:42,373] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
